README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
This is a great discovery on your part, I wasn't aware that the "clock" api could be used in this way to synchronized all nodes in a system.  I think this is the next logical step as not addressing this issue firsthand will hinder our ability to make future improvements given that ros is very time dependent.  Now, getting all nodes synced up might be rather involved task as there are lots of nodes, so I'd suggest that we address this in it's own PR.  Eventually, we could work on the _on demand process launcher_  feature after we address the sync problem.